x-pack/plugins/session_view/public/components/process_tree_node/index.tsx

```diff
@@ -128,6 +128,7 @@ export function ProcessTreeNode({
     ? 'sessionView:processTreeNodeExecIcon'
     : 'sessionView:processTreeNodeForkIcon';
   const processIcon = hasExec ? 'console' : 'branch';
+  const timeStampsNormal = moment(process.getDetails().process.start).format('MMM DD, YYYY @ hh:mm:ss.SSS');
```

Comment:
Hopefully there is a util for this standard format, if not we should make this a constant in constants.ts

Reply:
I'll confirm with the product/design folks, but I think we will need to create a component identical to `<PreferenceFormattedDate />` that is on security_solution, it uses date format from defined on the settings and collects the user browser timezone.

If that comes to be the case I can push a simplified version of that component on this PR for us to start using it